feeds/wlan-ap/wlan-ap-config/files/etc/hotplug.d/iface/40-fix-lan-wan-ip-conflict

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -x
+exec >>/root/hotplug.log 2>&1
+
+[ "$ACTION" = ifup -o "$ACTION" = ifupdate ] || exit 0
+[ "$INTERFACE" = wan ] || exit 0
+
+conflict=0
+wan_ipaddr="$(ubus call network.interface.wan status | grep \"address\" | grep -oE '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')"
```

Comment:
Can there be a delay between interface being up and IP address is received on the same?

Reply:
Interface br-wan is using DHCP.  Not sure  if it will come up without an IP address.
The code also looks at ifupdate events which I believe will catch it anyway.